### DISCUSSION TITLE
Import Copy:No Write:Yes Quick Question

### DISCUSSION THREAD

**Discussion by mkmtrading** [beetbox/beets#5498] [Q&A]
I’m assuming that Copy:No Write:Yes (update my music library in place where it currently resides) makes the PATH and TEMPLATE configuration stuff irrelevant, correct!

BTW beets is UNBELIEVABLE, thanks to all of you who’ve devoted countless man years to developing this project. I’m still using iTunes to synch 25K songs to an iPhone 16 with 1TB (hence the reason for Copy:No for right now) and am just now discovering the wonderful world of MusicBrainx, Picard and Beets. I hope to escape the restrictive Apple Music proprietary environment at some point and come over to the Android “dark side” but first I need to find a decent iTunes replacement

**Comment by snejus** [comment]
> I’m assuming that Copy:No Write:Yes (update my music library in place where it currently resides) makes the PATH and TEMPLATE configuration stuff irrelevant, correct!

Make sure that you've also got `write: no` for your use case!

**Reply by mkmtrading** [reply]
But if I want beets to update tag data in my iTunes library don’t I want write : yes?

**Comment by snejus** [comment]
Apologies!!! I meant `move: no` 😅

**Reply by mkmtrading** [reply]
No need to apologize, I do not have move:no in my config.yaml file but I just added it for safe measure; nothing got moved so the default with copy:no must be move:no
